Geometry/VeryForwardData/data/CTPPS_Pixel_2021/Modules/v1/ppstrackerMaterials.xml

```diff
@@ -29,42 +29,12 @@
    </MaterialFraction>
   </CompositeMaterial>
 
-  <CompositeMaterial name="Pix_Fwd_Bump" density="0.41391*g/cm3" symbol=" " method="mixture by weight">
```

Comment:
@cvuosalo , is it OK that you fix this file and not create a new one v2?

Reply:
Just linking here the [comment](https://github.com/cms-sw/cmssw/pull/34974#issuecomment-903411354) from @cvuosalo that says " a new version of `ppstrackerMaterials.xml` is not needed because this file has not yet been uploaded to the DB"